core/src/test/java/org/mule/util/TestsLogConfigurationHelper.java

```diff
@@ -63,7 +65,17 @@ public static void clearLoggingConfig()
 
     private static String findLogConfigurationPath(Class<?> testClass)
     {
-        File folder = new File(testClass.getClassLoader().getResource("").getPath().toString());
+        String encodedFolder = testClass.getClassLoader().getResource("").getPath().toString();
+
+        String folderPath ;
+        try {
+            folderPath = URLDecoder.decode(encodedFolder, "UTF-8");
+        } catch (UnsupportedEncodingException e) {
+            folderPath = encodedFolder;
```

Comment:
this is wrong... is you can't decode it you can't just return a path which might potentially have wrong characters. If this case you should return NULL_CONFIG_FILE